itests/eth_block_hash_test.go

```diff
@@ -65,18 +65,26 @@ func TestEthBlockHashesCorrect_MultiBlockTipset(t *testing.T) {
 		require.NoError(t, err)
 
 		ethBlockA, err := n2.EthGetBlockByNumber(ctx, hex, true)
-		// Cannot use static ErrFullRound error for comparison since it gets reserialized as a JSON RPC error.
-		if err != nil && strings.Contains(err.Error(), "null round") {
-			require.Less(t, ts.Height(), abi.ChainEpoch(i), "did not expect a tipset at epoch %d", i)
+		if err != nil {
+			if strings.Contains(err.Error(), "null round") {
```

Comment:
why have we changed this code ?

Reply:
Oh sorry didn't mean to push this, I was testing out something else